integration-tests/container/processes-listening-on-ports/process-listening-on-ports.c

```diff
@@ -0,0 +1,111 @@
+#include <netinet/in.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include <string.h>
+
+int openPort(int port) {
+	int server_fd;
+	struct sockaddr_in address;
+	int opt = 1;
+
+	if ((server_fd = socket(AF_INET, SOCK_STREAM, 0)) < 0) {
+		perror("socket failed");
+		exit(EXIT_FAILURE);
+	}
+
+	if (setsockopt(server_fd, SOL_SOCKET,
+				SO_REUSEADDR | SO_REUSEPORT, &opt,
+				sizeof(opt))) {
+		perror("setsockopt");
+		exit(EXIT_FAILURE);
+	}
+	address.sin_family = AF_INET;
+	address.sin_addr.s_addr = INADDR_ANY;
+	address.sin_port = htons(port);
+
+	if (bind(server_fd, (struct sockaddr*)&address,
+			sizeof(address))
+		< 0) {
+		perror("bind failed");
+		exit(EXIT_FAILURE);
+	}
+	if (listen(server_fd, 3) < 0) {
+		perror("listen");
+		exit(EXIT_FAILURE);
+	}
+
+	return server_fd;
+}
+
+void closePort(int server_fd) {
+	shutdown(server_fd, SHUT_RDWR);
```

Comment:
```suggestion
	close(server_fd);
```
Although it doesn't make a big difference, shutdown won't close the socket.

Reply:
Done